dev/devicelab/bin/tasks/platform_services__build.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+Future<Null> main() async {
+  await task(createPlatformServicesBuildTest());
```

Comment:
You don't need a build test unless you are interested in code size and build speed metrics. The start-up test has to build anyway and therefore will make sure that we can build the example app successfully.